source/globalCommands.py

```diff
@@ -4280,16 +4280,19 @@ def script_reportLinkDestination(
 		positioned on a link, or an element with an included link such as a graphic.
 		:param forceBrowseable: skips the press once check, and displays the browseableMessage version.
 		"""
+		focus = api.getFocusObject()
 		try:
 			ti: textInfos.TextInfo = api.getCaretPosition()
+			link = ti._getLinkDataAtCaretPosition()
```

Comment:
is this expected to raise a RuntimeError? If not, it should be moved to an `else` block below

Reply:
Fixed in b019bd162ce7778948c9ec89ff62a6e9cea976b7